scripts/i18n-keysets-clear copy.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
looks like it's better to change this filename

Reply:
There is already such file